src/Service/S3/composer.json

```diff
@@ -16,7 +16,7 @@
         "ext-dom": "*",
         "ext-filter": "*",
         "ext-hash": "*",
-        "async-aws/core": "^1.2"
+        "async-aws/core": "^1.8"
```

Comment:
Really? S3 is not compatible with version above 1.8? Why prefer-lowest test never failed?